demisto_sdk/commands/format/update_incidentfields.py

```diff
@@ -0,0 +1,33 @@
+from demisto_sdk.commands.common.tools import print_color, LOG_COLORS
+from demisto_sdk.commands.format.update_generic_json import BaseUpdateJSON
+from demisto_sdk.commands.common.hook_validations.incident_field import IncidentFieldValidator
+
+ARGUMENTS_DEFAULT_VALUES = {
```

Comment:
lets think of a way to have this at only one place. since if a field will have to be added, we do not want to add it in all the `update_<json_entity>.py` files.